api_documentation.md

```diff
@@ -0,0 +1,57 @@
+# API Documentation
+
+> Note: all endpoints return JSON
+
+### /api/login
+*method:* **POST**  
+*parameters:* **username**, **sim_id**, **password**  
+*description:* creates a user entry in the database and compares the entered password with the simulation password
+*onsuccess returns:* the user model  
+
+### /api/simulations
+*method:* **GET**  
+*parameters:* **none**  
+*description:* fetches all the simulations
+*onsuccess returns:* list of simulations in the database  
```

Comment:
Some of these endpoints have the "onsuccess returns" on a line by itself, but some (this endpoint is the first) have the "onsuccess return" text on the same line as the description:
![image](https://cloud.githubusercontent.com/assets/449185/6117892/bb51e080-b07d-11e4-9a9e-097063e411b3.png)